chrome-launcher/README.md

```diff
@@ -97,3 +97,22 @@ chromeLauncher.launch({
   console.log(`Chrome debugging port running on ${chrome.port}`);
 });
 ```
+
+### Continuous Integration
+
+Unlike development environment, on CI environment Chrome binary is not present. 
```

Comment:
yikes really need #2510 right about now huh ;) maybe pin to a specific version instead of master since we change things?